From 80cc04091410d6a322fee1a2922fdf867395f00a Mon Sep 17 00:00:00 2001
From: Patrick Ohly <patrick.ohly@intel.com>
Date: Fri, 29 May 2015 10:21:57 +0200
Subject: [PATCH] work around gmock pthread dependency

In meta-oe, gmock's .pc file does not declare that users of
gmock must link against pthread. Let's work around that
here by always linking tests against libpthread.

Upstream-status: Inappropriate [embedded specific]

Signed-off-by: Patrick Ohly <patrick.ohly@intel.com>
---
 test/CMakeLists.txt | 1 +
 1 file changed, 1 insertion(+)

diff --git a/test/CMakeLists.txt b/test/CMakeLists.txt
index 25a70db..f490a24 100644
--- a/test/CMakeLists.txt
+++ b/test/CMakeLists.txt
@@ -138,6 +138,7 @@ ADD_EXECUTABLE(${TARGET_CYNARA_TESTS}
 TARGET_LINK_LIBRARIES(${TARGET_CYNARA_TESTS}
     ${PKGS_LDFLAGS}
     ${PKGS_LIBRARIES}
+    pthread
     crypt
 )
 INSTALL(TARGETS ${TARGET_CYNARA_TESTS} DESTINATION ${BIN_INSTALL_DIR})
-- 
2.1.4

